playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/pre.yml

```diff
@@ -100,8 +100,8 @@
   hosts: oo_masters_to_config:oo_nodes_to_config
   vars:
     target_version: "{{ '1.2' if deployment_type == 'origin' else '3.1.1.900' }}"
-    openshift_docker_hosted_registry_insecure: True
-    openshift_docker_hosted_registry_network: "{{ hostvars[groups.oo_first_master.0].openshift.common.portal_net }}"
+    docker_hosted_registry_insecure: "{{ openshift_docker_hosted_registry_insecure | default(True) }}"
```

Comment:
Can we not continue to set it like we have been? Everywhere we're including a role that includes `openshift_docker` from a playbook, set

```
openshift_docker_hosted_registry_network: "{{ hostvars[groups.oo_first_master.0].openshift.common.portal_net }}"
```

WRT registry insecure, we would set like below within `openshift_docker_facts` and within `openshift_facts` library default the value to `True` allowing the `openshift_docker_facts` to override.

```
openshift_facts:
  role: docker
  local_facts:
    hosted_registry_insecure: "{{ openshift_docker_hosted_registry_insecure | default(None) }}"
```